server/services/participants.js

```diff
@@ -70,7 +78,7 @@ const getParticipants = async (user, pagination, sortField,
     : {
       ...(regionFilter && user.regions.includes(regionFilter))
         ? { 'body.preferredLocation ilike': `%${regionFilter}%` }
-        : { and: [userRegionQuery(user.regions, 'body.preferredLocation')] },
+        : { or: [{ and: [userRegionQuery(user.regions, 'body.preferredLocation')] }] },
```

Comment:
Why is there an `and` nested in there with a single condition?

Reply:
Oh got it. This is appended to (conditonally) on line 130. Maybe make a comment about this?